tests/authz-wildcard2.json

```diff
@@ -0,0 +1,761 @@
+            }
+        },
+        {
+            "name": "assignOrganizationOrgAAdminToUserAdminA",
+            "request": {
+                "method": "POST",
+                "url": "/v1/warrants",
+                "body": {
+                    "objectType": "organization",
+                    "objectId": "org-A",
+                    "relation": "admin",
+                    "subject": {
+                        "objectType": "user",
+                        "objectId": "admin-A"
+                    }
+                }
+            },
+            "expectedResponse": {
+                "statusCode": 200,
+                "body": {
+                    "objectType": "organization",
+                    "objectId": "org-A",
+                    "relation": "admin",
+                    "subject": {
+                        "objectType": "user",
+                        "objectId": "admin-A"
+                    }
+                }
+            }
+        },
+        {
+            "name": "assignOrganizationOrgBAdminToUserAdminB",
+            "request": {
+                "method": "POST",
+                "url": "/v1/warrants",
+                "body": {
+                    "objectType": "organization",
+                    "objectId": "org-B",
+                    "relation": "admin",
+                    "subject": {
+                        "objectType": "user",
+                        "objectId": "admin-B"
+                    }
+                }
+            },
+            "expectedResponse": {
+                "statusCode": 200,
+                "body": {
+                    "objectType": "organization",
+                    "objectId": "org-B",
+                    "relation": "admin",
+                    "subject": {
+                        "objectType": "user",
+                        "objectId": "admin-B"
+                    }
+                }
+            }
+        },
+        {
+            "name": "assignScheduleScheduleA1ParentToDepartmentDeptA1",
+            "request": {
+                "method": "POST",
+                "url": "/v1/warrants",
+                "body": {
+                    "objectType": "schedule",
+                    "objectId": "schedule-A1",
+                    "relation": "parent",
+                    "subject": {
+                        "objectType": "department",
+                        "objectId": "dept-A1"
+                    }
+                }
+            },
+            "expectedResponse": {
+                "statusCode": 200,
+                "body": {
+                    "objectType": "schedule",
+                    "objectId": "schedule-A1",
+                    "relation": "parent",
+                    "subject": {
+                        "objectType": "department",
+                        "objectId": "dept-A1"
+                    }
+                }
+            }
+        },
+        {
+            "name": "assignScheduleScheduleA1ViewerToUserExternalUser1",
```

Comment:
Are we testing anything related to this warrant for `external-user-1`?

Reply:
No, but I'll add a test case for it just to test the direct match case